specs/Cargo.toml

```diff
@@ -1,2 +1,3 @@
 [workspace]
 members = ["hacspec-lib", "kyber"]
+resolver = "1"
```

Comment:
This shouldn't really do anything because it's the default. Only 2 will set it to 2.